### PR TITLE
SUPER Initial Yield Offering

### PR DESCRIPTION
**Pull request by shahzaibak**

### CODE REVIEW

**Comment by davidcuellarrdx** [comment]
- Due to the wipe of stokenet, please redeploy the packages and update your dApp.
- Please update your PR integrating all the files and folders in only 1 PR. All files and folders should be inside your folder and not in the main challenge folder (8-yield-derivatives)
- Frontend README file is the same as the https://github.com/radixdlt/official-examples/tree/main/getting-started/react-js-dapp , please update it with YOUR main frontend components
- Please create your main README files with detailed information explaining: 
    - The dApp information
    - The scrypto package’s functions
    - How to use the Scrypto packages
    - How to run the dApp
    - How to use Transaction Manifests in Stokenet (add addresses needed).
    - Add MIT License at the end of the README: https://github.com/radixdlt/scrypto-challenges?tab=readme-ov-file#license 
- Please fix all transaction errors in the dApp
- Please update all your code (scrypto and dapp) with comments to understand how it works.
- [Optional]: Deploy your dApp using github pages or netlify and add the link in the README file.

**Comment by shahzaibak** [comment]
Folders have only been consolidated to one PR - readme and code comments are being done now

File: 8-yield-derivatives/README.md
Comment (davidcuellarrdx) [inline 8-yield-derivatives/README.md:1]: Hello, this README.md shouldn't be deleted

File: 8-yield-derivatives/SUPER/Website/Front End/README.md
Comment (davidcuellarrdx) [inline 8-yield-derivatives/SUPER/Website/Front End/README.md:1]: This is the same react-js-dapp README file. Please update it with your functions.

File: 8-yield-derivatives/README.md
Comment (shahzaibak) [reply 8-yield-derivatives/README.md:1]: oops sorry it was an accident, I'll bring it back!

File: 8-yield-derivatives/SUPER/README.md
Comment (davidcuellarrdx) [inline 8-yield-derivatives/SUPER/README.md:1]: Please review hyperlinks, some of them are not working. Please add MIT license.

**Review by davidcuellarrdx — COMMENTED** [review]
Please check comments.

File: 8-yield-derivatives/SUPER/Smart Contract/tests/lib.rs
Comment (davidcuellarrdx) [inline 8-yield-derivatives/SUPER/Smart Contract/tests/lib.rs:1]: Please remove file

File: 8-yield-derivatives/SUPER/Smart Contract/README.md
Comment (davidcuellarrdx) [inline 8-yield-derivatives/SUPER/Smart Contract/README.md:1]: Please update README file explaining "How to use the Scrypto packages". For example: run `resim reset`, run `resim new-account`, run `resim publish .`
to instantiate `resim run new.rtm` , etc...

**Review by davidcuellarrdx — COMMENTED** [review]
Please review comments

File: 8-yield-derivatives/SUPER/Smart Contract/README.md
Comment (shahzaibak) [reply 8-yield-derivatives/SUPER/Smart Contract/README.md:1]: done!

**Pull request merged**